libs/shared/src/containers/UploadWallet/index.tsx

```diff
@@ -1,12 +1,11 @@
 import * as React from 'react';
-import { Label, Container, Header, Grid } from 'semantic-ui-react';
+import { Container, Header } from 'semantic-ui-react';
 import { IsValidAddress } from '@thecointech/utilities';
 import styles from './styles.module.less';
 import { defineMessages, FormattedMessage } from 'react-intl';
-import { AccountMap } from '../AccountMap';
-import { useHistory } from 'react-router';
-
-import illustration from "./images/illust_flowers.svg";
+import {useDropzone, FileRejection} from 'react-dropzone';
```

Comment:
Our drop-zone wasn't working for actual drops.  Switched it out for a published version, yay open-source